custom-tsdb/values-primary.yaml

```diff
@@ -18,8 +18,12 @@ global:
     queryService: http://thanos-query-frontend.thanosnamespace.svc:9090 # the location of the aggregated metrics. For thanos, thanos-query-frontend is recommended
 # serviceMonitor: # if your prometheus supports serviceMonitors, set these to true. Otherwise see extraScrapeConfigs.yaml in this directory
 #   enabled: true # note that you must also apply the network-costs-serviceMonitor.yaml if you enable this.
+#   additionalLabels:
+#     label-key: label-value
 #   networkCosts: # this is waiting for a fix that is coming in kubecost 1.98. Until then, see the network-costs-serviceMonitor.yaml in this folder
```

Comment:
Any changes that can be made here?

Reply:
you read it more carefully than me, per usual! thx